### PR TITLE
[Docs] Replace play.clickhouse.com urls with sql.clickhouse.com

### PR DESCRIPTION
**Pull request by lio-p**
Changelog category (leave one):

- Documentation (changelog entry is not required)

This PR find and replaces all URLs pointing to the old Playground to migrate to the new experience. 

Some URLs are also removed as they pointed to queries that won't execute. 

The changelogs docs remains untouched.  

-- PR pending deployment of https://sql.clickhouse.com/

### CODE REVIEW

File: docs/en/development/contrib.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ SELECT library_name, license_type, license_path FROM system.licenses ORDER BY li
 Note that the listed libraries are the ones located in the `contrib/` directory of the ClickHouse repository.
 Depending on the build options, some of the libraries may have not been compiled, and, as a result, their functionality may not be available at runtime.
 
-[Example](https://play.clickhouse.com/play?user=play#U0VMRUNUIGxpYnJhcnlfbmFtZSwgbGljZW5zZV90eXBlLCBsaWNlbnNlX3BhdGggRlJPTSBzeXN0ZW0ubGljZW5zZXMgT1JERVIgQlkgbGlicmFyeV9uYW1lIENPTExBVEUgJ2VuJw==)
+[Example](https://sql.clickhouse.com/play?query_id=478GCPU7LRTSZJBNY3EJT3)
```
Comment (vdimir) [inline docs/en/development/contrib.md:21]: Query is missing for some reason

image

**Review by vdimir — DISMISSED** [review]

**Comment by robot-clickhouse-ci-1** [comment]
*This is an automated comment for commit 83dda28f59248d50064d0d0405b9a708202c8ed9 with description of existing statuses. It's updated for the latest CI running*

[✅ Click here](https://s3.amazonaws.com/clickhouse-test-reports/70953/83dda28f59248d50064d0d0405b9a708202c8ed9/ci_running.html) to open a full report in a separate page

<details><summary>Successful checks</summary>
<table>
<thead><tr><th>Check name</th><th>Description</th><th>Status</th></tr></thead>
<tbody>
<tr><td>Style check</td><td>Runs a set of checks to keep the code style clean. If some of tests failed, see the related log from the report</td><td><a href="https://s3.amazonaws.com/clickhouse-test-reports/70953/83dda28f59248d50064d0d0405b9a708202c8ed9/style_check.html">✅ success</a></td></tr>
<tbody>
</table>
</details>

**Comment by alexey-milovidov** [comment]
A bit premature, because it isn't released yet: https://sql.clickhouse.com/

**Comment by lio-p** [comment]
Added a comment in the PR description. 

We're waiting to have https://sql.clickhouse.com/ live before merging. Just wanted to have the PR ready to go beforehand.

File: docs/en/development/contrib.md (hunk 1 above)
Comment (gingerwizard) [reply docs/en/development/contrib.md:21]: @vdimir this is pending a DNS deployment of a new UI, hence draft

**Review by justindeguzman — APPROVED** [review]

**Review by justindeguzman — APPROVED** [review]

**Pull request merged**